src/features/DebugSession.ts

```diff
@@ -1,96 +1,96 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import vscode = require("vscode");
+import vscode from "vscode";
```

Comment:
```suggestion
```

Reply:
I think there was a reason it was doubled up here, let me check, but it might just be an esmoduleinterop thing.

Reply:
So the reason was that I didn't want to "touch" the code that wasn't mine. Since you asked I went ahead and updated all the references. Commited ce23551c70829e36c5f7be1a50f198caaa5aa3a8

Reply:
I like it!